r/tools/nixlibs.R

```diff
@@ -213,7 +213,7 @@ read_system_release <- function() {
 
 find_available_binary <- function(os) {
   # Download a csv that maps one to the other, columns "actual" and "use_this"
-  u <- "https://raw.githubusercontent.com/ursa-labs/arrow-r-nightly/master/linux/distro-map.csv"
+  u <- "https://raw.githubusercontent.com/apache/arrow/master/r/tools/distro-map.csv"
   lookup <- try(utils::read.csv(u, stringsAsFactors = FALSE), silent = quietly)
```

Comment:
Since the file is included in the package now, you could fall back to using the local copy. (I still think it is good to try to download since we can update the one on master.)